docs/readmes/lte/configure_sentry.md

```diff
@@ -46,27 +47,44 @@ The follwing script outlines the necessary steps.
 
 ```bash
 #!/bin/bash
-# To install sentry-cli, run `curl -sL https://sentry.io/get-cli/ | bash`
 
-TODO: change me!
-EXECS_PATHS=""
+### TODO: fill in all variables!
```

Comment:
TODO here isn't totally clear to me - is this "Magma needs to modify script TODO" or "you the copy paster of this script needs TODO the following"? If it's the latter, why did we not have `TODO: fill in all variables` up above on e.g. sentry_url_python.